source/dub/dub.d

```diff
@@ -273,36 +261,44 @@ class Dub {
 
 	private void init(NativePath root_path)
 	{
+		import configy.Read;
+
 		this.m_dirs = SpecialDirs.make();
 
-		m_config = new DubConfig(jsonFromFile(m_dirs.systemSettings ~ "settings.json", true), m_config);
+		const systemSettings = (m_dirs.systemSettings ~ "settings.json").toNativeString();
+		if (systemSettings.exists)
+			this.m_config = this.m_config.merge(
+				parseConfigFile!UserConfiguration(CLIArgs(systemSettings)));
```

Comment:
please add a helper function to call for each additional config check + load

Reply:
Done